setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
getting the requirements install locally is the same as before, particularly because I added the `requirements.txt => requirements/base.txt` alias.

but, for conciseness, you can install a local wizard builder version with

```bash
# location: another package in the same folder as wizard builder
pip uninstall wizard_builder
pip install -e ../wizard_builder
```